kubejs/server_scripts/constants/sandstones.js

```diff
@@ -0,0 +1,13 @@
+const sandstones = [
```

Comment:
Since you're using this is other files, you need to make sure this file loads first. Adding `//priority: 1010` as the very first line should do the trick

(in our case, any number over 1 works, since we don't use priority a lot)